composer.json

```diff
@@ -51,7 +51,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "0.8-dev"
+            "dev-master": "1.1-dev"
```

Comment:
Please don't mix preparation to 1.0 release with preparation for 1.1 development.
In the middle of that (after release 1.0, before starting development 1.1) you will need to create separate branch (eg `1.0`) for which you will merge bugfixes, and then merge 1.0 to master.